.github/workflows/lint-code.yml

```diff
@@ -58,3 +58,29 @@ jobs:
             Thanks again for your contribution!
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           allow-repeats: false
+
+  Isort:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source-code repository
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install python dependencies
+        run: |
+          python -m pip install --upgrade pip -r requirements-dev.txt
+          pip install -e .
+
+      - name: Install isort
+        run: pip install isort
+
+      - name: python-isort
+        uses: isort/isort-action@v0.1.0
+        with:
+          isortVersion: "latest"
+          configuration: "--profile black"
```

Comment:
I suspect that we can leave all configuration to `pyproject.toml`. That'd be better, as then the same config will be used when running isort outside of the action.